scripts/generators/helper_file_generator.py

```diff
@@ -447,16 +448,23 @@ def DeclareCounter(self, string_var, declare_flag):
     #
     # Combine safe struct helper header file preamble with body text and return
     def GenerateSafeStructHelperHeader(self):
-        safe_struct_helper_header = '\n'
-        safe_struct_helper_header += '#pragma once\n'
-        safe_struct_helper_header += '#include <vulkan/vulkan.h>\n'
-        safe_struct_helper_header += '#include <stdlib.h>\n'
-        safe_struct_helper_header += '#include <algorithm>\n'
-        safe_struct_helper_header += '\n'
-        safe_struct_helper_header += 'void *SafePnextCopy(const void *pNext);\n'
-        safe_struct_helper_header += 'void FreePnextChain(const void *pNext);\n'
-        safe_struct_helper_header += 'char *SafeStringCopy(const char *in_string);\n'
-        safe_struct_helper_header += '\n'
+        safe_struct_helper_header = '''
+#pragma once
+#include <vulkan/vulkan.h>
+#include <stdlib.h>
```

Comment:
Use cstdlib instead of stdlib.h

```suggestion
#include <cstdlib>
```